dictionary/homework_dictionary.py

```diff
@@ -0,0 +1,36 @@
+"""Solution task 1"""
+check_name = input("Please, enter your name:  ").title()
+roles_dict = {
+    'admin': ['George', 'Ruslan', 'Valeriy', 'Irina'],
+    'maintainer': ['Julia', 'Alex'],
+    'manager': ['Piter', 'Max', 'Sophia'],
+    'developer': ['Natali', 'Nikita', 'Diana']
+}
+
+for role, list_names in roles_dict.items():
+    if check_name in list_names:
+        print(f"Hello {role}")
+        break
+    else:
```

Comment:
block else should be added for loop, not for if. Block else for loops work if block break didn't work